terraform/gcp/modules/rekor/variables.tf

```diff
@@ -120,3 +120,15 @@ variable "gcp_lb_traffic_weight" {
   type        = number
   default     = 0
 }
+
+variable "new_entry_pubsub_topic" {
+  description = "name of the pubsub topic for new entries added to rekor"
+  type        = string
+  default     = "new-entry"
+}
+
+variable "new_entry_pubsub_consumers" {
+  description = "IAM members that can consume messages from the topic"
+  type        = list(string)
+  default     = ["allAuthenticatedUsers"]
```

Comment:
I would have the default be restrictive rather permissive, else private deployments will inadvertently publicize their entries.

Reply:
Makes sense, updated.